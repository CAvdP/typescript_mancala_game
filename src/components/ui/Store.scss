.store {
    border-radius: 10px;
    height: 200px;
    width: 200px;
    margin: 20px;
    font-size: 6rem;
    display: flex;
    align-items: center;
    justify-content: center;
}