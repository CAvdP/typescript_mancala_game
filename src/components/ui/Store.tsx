import React from 'react';
import './Store.scss';

interface Props{
    weight:number;
    className:string;
}

function Store({weight,className}: Props){
return (
    <div className={`store ${className}`}>{weight}</div>
)
}

export default Store