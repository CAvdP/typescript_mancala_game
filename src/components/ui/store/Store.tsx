import React from 'react';
import './styling/Store.scss';

interface Props{
    weight:number;
    enabled:boolean;
    className:string;
}

function Store({weight, enabled,className}: Props){
return (
    <div className={`store ${enabled === true ? className: 'disabled'}`}>{weight}</div>
)
}

export default Store