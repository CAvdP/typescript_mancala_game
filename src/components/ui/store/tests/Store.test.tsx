import React from 'react'
import renderer from 'react-test-renderer'
import Store from '../Store';



describe("Pit Component", () => {
    it('matches the saved snapshot', () => {
        const actualSnapshot = renderer
          .create(<Store weight={0} enabled={false} className={'neon_pink'} />)
          .toJSON()
          
        expect(actualSnapshot).toMatchSnapshot()
      });
});