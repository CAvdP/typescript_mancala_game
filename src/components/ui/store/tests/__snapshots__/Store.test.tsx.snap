// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pit Component matches the saved snapshot 1`] = `
<div
  className="store disabled"
>
  0
</div>
`;
